independent-projects/enforcer-rules/pom.xml

```diff
@@ -91,6 +91,7 @@
                 <version>${maven-invoker-plugin.version}</version>
                 <!-- https://maven.apache.org/plugins/maven-invoker-plugin/usage.html -->
                 <configuration>
+                    <mavenOpts>-Dscan=false</mavenOpts>
```

Comment:
why set it here?

Reply:
Some [smoke tests](https://github.com/quarkusio/quarkus/tree/main/independent-projects/enforcer-rules/src/it/smoketest) are triggering some [Maven builds](https://github.com/quarkusio/quarkus/blob/main/independent-projects/enforcer-rules/src/it/smoketest/invoker.properties) which would publish some build scans.